nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -994,6 +1031,257 @@ After going through this checklist, you're ready to go!
+				flight_state = NO_APU
+		if("apu")
+			if(!throttle_lock || flight_state >= APU_SPUN)
+				to_chat(usr, "You can't flip this switch without first engaging the throttle lock or when in flight.</span>")
+				return
+			if(flight_state < NO_APU)
+				to_chat(usr, "You can't flip this switch without first engaging the battery.</span>")
+				return
+			to_chat(usr, "You flip the APU switch.</span>")
+			flight_state = APU_SPUN
+			playsound(src, 'nsv13/sound/effects/fighters/apu_start.ogg', 100, FALSE)
+			addtimer(VARSET_CALLBACK(src, warmup_cooldown, FALSE), 15 SECONDS)
+			addtimer(CALLBACK(src, .proc/check_start), 16 SECONDS) //Throttle up now....
+			return
+		if("throttle_lock")
+			to_chat(usr, "You flip the throttle lock switch.</span>")
+			throttle_lock = !throttle_lock
+		if("shutdown")
+			if(!throttle_lock)
+				to_chat(usr, "You cannot shut down [src]'s engines without first engaging the throttle lock.</span>")
+				return
+			to_chat(usr, "You start flipping switches and perfoming a controlled shutdown...</span>")
+			relay('nsv13/sound/effects/fighters/powerswitch.ogg')
+			if(do_after(usr, 5 SECONDS, target=src))
+				flight_state = NO_IGNITION
+				playsound(src, 'nsv13/sound/effects/ship/rcs.ogg', 100, TRUE)
+				visible_message("<span class='warning'>[src]'s engine fizzles out!</span>")
+				stop_relay(CHANNEL_SHIP_ALERT)
+		if("canopy_lock")
+			toggle_canopy()
+		if("eject")
+			if(!ejecting)
+				to_chat(usr, "<span class='notice'>WARNING AUTO-EJECT SEQUENCE COMMENCING IN T-5 SECONDS. USE THIS SWITCH AGAIN TO CANCEL THIS ACTION.</span>")
+				relay('nsv13/sound/effects/fighters/switch.ogg')
+				relay('nsv13/sound/effects/ship/general_quarters.ogg')
+				addtimer(CALLBACK(src, .proc/eject), 10 SECONDS)
+				ejecting = TRUE
+				return
+			else
+				to_chat(usr, "<span class='notice'>WARNING AUTO-EJECT SEQUENCE CANCELLED.</span>")
+				relay('nsv13/sound/effects/fighters/switch.ogg')
+				ejecting = FALSE
+				return
+
+		if("docking_mode")
+			to_chat(usr, "<span class='notice'>You [docking_mode ? "disengage" : "engage"] [src]'s docking computer.</span>")
+			docking_mode = !docking_mode
+			relay('nsv13/sound/effects/fighters/switch.ogg')
+			return //Dodge the cooldown because these actions should be instant
+		if("brakes")
+			toggle_brakes()
+			relay('nsv13/sound/effects/fighters/switch.ogg')
+			return //Dodge the cooldown because these actions should be instant
+		if("weapon_safety")
+			toggle_safety()
+			relay('nsv13/sound/effects/fighters/switch.ogg')
+			return //Dodge the cooldown because these actions should be instant
+	warmup_cooldown = TRUE
+	addtimer(VARSET_CALLBACK(src, warmup_cooldown, FALSE), 3 SECONDS)
+	relay('nsv13/sound/effects/fighters/switch.ogg')
+
+/obj/structure/overmap/fighter/proc/get_fuel()
+	var/obj/item/twohanded/required/fighter_component/fuel_tank/sft = get_part(/obj/item/twohanded/required/fighter_component/fuel_tank)
+	if(!sft)
+		return 0

```

Comment:
FALSE

Reply:
Actually, since this is supposed to be returning the number of units of fuel, 0 is good here